src/components/Committees/CommitteesBanner/CommitteesBanner.js

```diff
@@ -0,0 +1,21 @@
+import React from 'react';
+
+export default class CommitteesBanner extends React.Component {
+	render() {
+		return (
+			<div className="committee-banner-container">
+				<div className="banner-main-content">
+					<div className="banner-left-col">
+						<img src={process.env.PUBLIC_URL + "/images/acm_committees.svg"} alt="the logos of all ACM committees" />
+					</div>
+					<div className="banner-right-col">
```

Comment:
I'm a little confused with this className, doesn't seem to be used in the associated `.scss` file

Reply:
Ah, originally had gone with a two-column layout for larger screen sizes but decided not to go that route; good catch